src/schema/account-number.ts

```diff
@@ -1,32 +1,32 @@
 import { JSONSchemaType } from 'ajv'
-import { MockCheckingAccount, FiatType } from '../types'
+import { AccountNumber, FiatAccountType } from '../types'
 
-export const mockCheckingAccountSchema: JSONSchemaType<MockCheckingAccount> = {
-  $id: 'MockCheckingAccountSchema',
+export const accountNumberSchema: JSONSchemaType<AccountNumber> = {
+  $id: 'AccountNumberSchema',
   type: 'object',
   properties: {
-    bankName: {
+    institutionName: {
       type: 'string',
     },
     accountName: {
       type: 'string',
     },
-    fiatType: {
+    fiatAccountType: {
       type: 'string',
-      enum: Object.values(FiatType),
+      enum: [FiatAccountType.BankAccount],
```

Comment:
this is kind of weird but I wasn't sure how to do it better, I welcome ideas though

Reply:
does `Object.values(FiatAccountType)` not work, like `fiatType` in the previous commit?

Reply:
I think only bank account is viable